src/main/java/org/moltimate/moltimatebackend/service/LigandService.java

```diff
@@ -61,31 +59,73 @@
         }
         EcNumberValidator.validate(ecNumber);
 
-
         log.info("Retrieving Ligands associated EC Class " + ecNumber);
-
-        //call method from the motifRepository to get the pdb ids of the motifs that
```

Comment:
Should these have been re-written instead of removed completely?

Reply:
Are you referring to the comments?